velox/functions/lib/window/NthValue.cpp

```diff
@@ -82,12 +87,24 @@ class NthValueFunction : public exec::WindowFunction {
     auto rawFrameStarts = frameStarts->as<vector_size_t>();
     auto rawFrameEnds = frameEnds->as<vector_size_t>();
 
+    vector_size_t nullsOffset = rawFrameStarts[0];
+    if (ignoreNulls_) {
+      if (validRows.hasSelections()) {
```

Comment:
When would this not be the case? Should we have empty validRows at the start of this method?

Reply:
This code is out of date now.